src/azure-cli/azure/cli/command_modules/cdn/_help.py

```diff
@@ -159,6 +159,293 @@
+    type: string
+    short-summary: Fragment to add to the redirect URL.
+  - name: --query-string-behavior
+    type: string
+    short-summary: Caching behavior for the requests. Possible values are Include, IncludeAll, Exclude and ExcludeAll.
+  - name: --query-parameters
+    type: string
+    short-summary: query parameters to include or exclude (comma separated).
+  - name: --source-pattern
+    type: string
+    short-summary: define a request URI pattern that identifies the type of requests that may be rewritten.
+  - name: --destination
+    type: string
+    short-summary: Define the destination path for be used in the rewrite. This will overwrite the source pattern.
+  - name: --preserve-unmatched-path
+    type: boolean
+    short-summary: If True, the remaining path after the source pattern will be appended to the new destination path.
+examples:
+  - name: Create a global rule to disable caching.
+    text: >
+        az cdn endpoint rule add -g group -n endpoint --profile-name profile --order 0\\
+            --rule-name global --action-name CacheExpiration --cache-behavior BypassCache
+  - name: Create a rule for http to https redirect
+    text: >
+        az cdn endpoint rule add -g group -n endpoint --profile-name profile --order 1\\
+            --rule-name "redirect" --match-variable RequestScheme --operator Equal --match-values HTTPS\\
+            --action-name "UrlRedirect" --redirect-protocol Https --redirect-type Moved
+"""
+
+helps['cdn endpoint rule remove'] = """
+type: command
+short-summary: Remove a delivery rule from an endpoint.
+parameters:
+  - name: --rule-name
+    type: string
+    short-summary: Name of the rule.
+examples:
+  - name: Remove the global rule.
+    text: >
+        az cdn endpoint rule remove -g group -n endpoint --profile-name profile --rule-name Global\\
+"""
+
+helps['cdn endpoint rule show'] = """
+type: command
+short-summary: show delivery rules asscociate with the endpoint.
+examples:
+  - name: show delivery rules asscociate with the endpoint.
+    text: >
+        az cdn endpoint rule show -g group --profile-name profile-name
+"""
+
+helps['cdn endpoint rule condition'] = """
+type: group
+short-summary: Manage delivery rule conditions for an endpoint.
+"""
+
+helps['cdn endpoint rule condition add'] = """
+type: command
+short-summary: Add a condition to a delivery rule.
+parameters:
+  - name: --rule-name
+    type: string
+    short-summary: Name of the rule.
+  - name: --match-variable
+    type: string
+    short-summary: Name of the match condition.
+  - name: --operator
+    type: string
+    short-summary: Operator of the match condition.
+  - name: --selector
+    type: string
+    short-summary: Selector of the match condition.
+  - name: --match-values
+    type: string
+    short-summary: Match values of the match condition (comma separated).
+  - name: --negate-condition
+    type: string
+    short-summary: Describes if this is negate condition or not.
```

Comment:
is the negate_condition a specific definition in CDN?

Reply:
this is actually a boolean argument. It can only be true or false